frontend/app/controllers/FrontPage.scala

```diff
@@ -8,101 +8,29 @@ trait FrontPage extends Controller {
   def index = CachedAction { implicit request =>
 
     val slideShowImages = Seq(
-      ResponsiveImageGroup(altText=Some("Guardian Live event: Pussy Riot - art, sex and disobedience"),
-        availableImages = Seq(
-          ResponsiveImage(
-            path="https://media.guim.co.uk/eab86e9c81414932e0d50a1cd609dccfc20ca5d2/0_0_2279_1368/2000.jpg",
-            width=2000
-          ),
-          ResponsiveImage(
-            path="https://media.guim.co.uk/eab86e9c81414932e0d50a1cd609dccfc20ca5d2/0_0_2279_1368/1000.jpg",
-            width=1000
-          ),
-          ResponsiveImage(
-            path="https://media.guim.co.uk/eab86e9c81414932e0d50a1cd609dccfc20ca5d2/0_0_2279_1368/500.jpg",
-            width=500
-          )
-        )
+      ResponsiveImageGroup(
+        altText=Some("Guardian Live event: Pussy Riot - art, sex and disobedience"),
+        availableImages=ResponsiveImageGenerator("eab86e9c81414932e0d50a1cd609dccfc20ca5d2/0_0_2279_1368", List(2000, 1000, 500))
       ),
-      ResponsiveImageGroup(altText=Some("Guardian Live with Russell Brand"),
-        availableImages = Seq(
-          ResponsiveImage(
-            path="https://media.guim.co.uk/da867c363957d748bd04cfd9d4890033203a58c6/0_0_2279_1368/2000.jpg",
-            width=2000
-          ),
-          ResponsiveImage(
-            path="https://media.guim.co.uk/da867c363957d748bd04cfd9d4890033203a58c6/0_0_2279_1368/1000.jpg",
-            width=1000
-          ),
-          ResponsiveImage(
-            path="https://media.guim.co.uk/da867c363957d748bd04cfd9d4890033203a58c6/0_0_2279_1368/500.jpg",
-            width=500
-          )
-        )
+      ResponsiveImageGroup(
+        altText=Some("Guardian Live with Russell Brand"),
+        availableImages=ResponsiveImageGenerator("da867c363957d748bd04cfd9d4890033203a58c6/0_0_2279_1368", List(2000, 1000, 500))
```

Comment:
Minor one but can the lists be defined on new line else we have to across to read them scroll.